### PR TITLE
Microsoft SSO and AD mention in callout box.

### PR DESCRIPTION
**Pull request by napcs**
## What does this PR do?

Add mention of Microsoft SSO and information about Google Workspace or Azure AD considerations.

### CODE REVIEW

**Review by flossypurse — APPROVED** [review]

File: docs-src/cloud/users-invite.md [hunk 1]
```diff
@@ -11,11 +11,14 @@ tags:
 ---
 
 :::caution
```
Comment (flossypurse) [inline docs-src/cloud/users-invite.md:13]: hmm... why is this a "caution" ?

just seems like an "info" callout -

**Review by rachfop — APPROVED** [review]

File: docs-src/cloud/users-invite.md (hunk 1 above)
Comment (napcs) [reply docs-src/cloud/users-invite.md:13]: It was originally a caution - changing seemed out of scope but open to changing.

File: docs-src/cloud/users-invite.md [hunk 2]
```diff
@@ -11,11 +11,14 @@ tags:
...
-Access to Temporal Cloud can be authorized via Google OAuth single sign-on or SAML, depending on your account setup.
+Access to Temporal Cloud can be authorized via Google Oath single sign-on, Microsoft single sign-on, or SAML, depending on your setup.
```
Comment (tminusplus) [inline docs-src/cloud/users-invite.md:14]: Google OAuth*

File: docs-src/cloud/users-invite.md [hunk 3]
```diff
@@ -11,11 +11,14 @@ tags:
...
 
 If you are using Google OAuth for single sign-on and an email address is not associated with a Google Account, the user must follow the instructions in the [Use an existing email address](https://support.google.com/accounts/answer/27441?hl=en#existingemail) section of [Create a Google Account](https://support.google.com/accounts/answer/27441).
 
 **Important:** Do _not_ create a Gmail account when creating a Google Account.
+
+If your organization uses Google Workspace or Microsoft Azure AD, and your IT administrator has enabled controls over single sign-on and permissions, then you will need to work with your IT administrator to allow logins to Temporal Cloud.
```
Comment (tminusplus) [inline docs-src/cloud/users-invite.md:20]: Could also be "single sign-on permissions".

**Review by tminusplus — APPROVED** [review]
Same comments for the account-setup/users.md doc 🚀

File: docs-src/cloud/users-invite.md (hunk 2 above)
Comment (napcs) [reply docs-src/cloud/users-invite.md:14]:
```suggestion
Access to Temporal Cloud can be authorized via Google OAuth single sign-on, Microsoft single sign-on, or SAML, depending on your setup.
```

File: docs-src/cloud/users-invite.md (hunk 3 above)
Comment (napcs) [reply docs-src/cloud/users-invite.md:20]:
```suggestion
If your organization uses Google Workspace or Microsoft Azure AD, and your IT administrator has enabled controls over single sign-on permissions, then you will need to work with your IT administrator to allow logins to Temporal Cloud.
```

**Pull request merged**